aiida/engine/processes/calcjobs/tasks.py

```diff
@@ -71,7 +71,7 @@ async def task_upload_job(process: 'CalcJob', transport_queue: TransportQueue, c
     initial_interval = get_config_option(RETRY_INTERVAL_OPTION)
     max_attempts = get_config_option(MAX_ATTEMPTS_OPTION)
 
-    authinfo = node.computer.get_authinfo(node.user)
+    authinfo = node.get_authinfo()
```

Comment:
this does somewhat change the code path - in `Node.get_authinfo` it goes via `AuthInfo.from_backend_entity` instead of using `Computer.get_authinfo` directly (which I think it should).

I think this should be fixed

Reply:
Indeed, for reference;

for `Computer.get_authinfo` it is:
`return AuthInfo.objects(self.backend).get(dbcomputer_id=self.id, aiidauser_id=user.id)`
but for `CalcJobNode.get_authinfo` it is:
`return AuthInfo.from_backend_entity(self.backend.authinfos.get(computer=computer, user=self.user))`

@sphuber you added the calcjob one, can you clarify if there is any reason not to replace it with: `return self.computer.get_authinfo(self.user)`

Reply:
ok I have changed this in 3344edc6cc1fdd98ab8720860af73a1ec1569f4e